website/src/components/fontPreloader/FontPreloader.js

```diff
@@ -224,10 +224,9 @@ export const FontPreloader = ({ title = '', ...rest }) => {
 					left: '10px',
 					width: '1px',
 					height: '1px',
-					padding: '10px',
 					overflow: 'hidden',
 					zIndex: 99999,
-					background: '#fff',
+					background: 'transparent',
 				}}
```

Comment:
Hey @dominikwilkowski can't we treat this like our VisuallyHidden component and use that component's styling?

Also visibility isn't a valid attribute... https://github.com/WestpacGEL/GEL/commit/c756e2fb877001e1e63e4568cca4ca8180a20590#r49242571